src/silly_sprites/opengl/buffer_object.cpp

```diff
@@ -0,0 +1,97 @@
+#include "buffer_object.hpp"
+
+namespace sly::gl {
+    void BufferObject::init() {
+        generate_vertex_array();
+        generate_vertex_buffer();
+        generate_element_buffer();
+
+        glVertexAttribPointer(0, 3, GL_FLOAT, GL_FALSE, 3 * sizeof(float), static_cast<void*>(0));
```

Comment:
Can't you just do this?
```suggestion
        glVertexAttribPointer(0, 3, GL_FLOAT, GL_FALSE, 3 * sizeof(float), nullptr);
```